index.html

```diff
@@ -1685,18 +1734,45 @@ <h2>Resolver Architectures</h2>
 			</figcaption>
 		</figure>
 
-		<div class="issue" data-number="28">
-			<p>TODO: Describe <a>local bindings</a> vs. <a>remote bindings</a>, and implications for privacy, security and trust.</p>
-			<p>Also describe mitigations against potential downsides of <a>remote bindings</a>, e.g.:</p>
-			<ul>
-				<li>A <a>DID resolver's</a> <a>remote binding</a> can use a trusted channel such as VPN or TLS with mutual authentication.</li>
-				<li>A <a>DID resolver</a> can add a data integrity proof (see [[DATA-INTEGRITY]])
-				to a <a>DID document</a> and/or the <a>DID resolution result</a>. Discuss what this does and doesn't achieve. Also see
-				<a href="https://www.w3.org/TR/did-core/#proving-control-and-binding">Proving Control and Binding</a>.</li>
-				<li>A client could query multiple <a>DID resolvers</a> over a <a>remote binding</a> and compare results.</li>
-			</ul>
-		</div>
-		<p class="issue">TODO: Discuss DID resolution in constrained user agents such as mobile apps and browsers.</p>
+        <p>
+            Whenever possible, <a>local bindings</a> are preferred, as they minimize dependencies on third parties
+            and intermediaries, reduce security risks, and maximize confidence in the integrity and correctness of the
+            results of the <a>DID resolution</a> and <a>DID URL dereferencing</a> functions.
+        </p>
+
+        <p>
+            In some cases, it might not be possible to use a <a>local binding</a>, for example in constrained IoT
```

Comment:
maybe a link to something IoT?
```suggestion
            In some cases, it might not be possible to use a <a>local binding</a>; for example, in constrained IoT (Internet of Things)
```